src/features/mangaViewer/components/SimilarTab.tsx

```diff
@@ -10,7 +10,7 @@ const SimilarTab = ({
 }) => {
   if (recommendations?.length === 0) {
     return (
-      <div className="h-full flex items-center justify-center"> NO manga</div>
+      <h1 className="h-full flex items-center justify-center"> NO manga</h1>
```

Comment:
For consistency purposes, maybe do "No" like above. Also, just check the spacing in front of "no" as mentioned above.